chatkit-core/src/main/kotlin/com/pusher/chatkit/ChatkitCore.kt

```diff
@@ -0,0 +1,77 @@
+package com.pusher.chatkit
+
+import com.pusher.chatkit.api.createPlatformClientModule
+import com.pusher.chatkit.state.createStoreModule
+import com.pusher.chatkit.users.api.createUserSubscriberModule
+import com.pusher.util.Result
+import com.pusher.util.asSuccess
+import elements.Error
+import org.koin.core.KoinApplication
+import org.koin.core.module.Module
+import org.koin.dsl.koinApplication
+import org.koin.dsl.module
+
+/**
+ * The entry point for JVM apps to connect with the backend service
```

Comment:
what do you mean by `JVM apps` - does this include typical android apps? If so would it be simpler to say `the entry point for apps to connect...`?

Reply:
Android apps don't run in JVM, they run in ART or Dalvik (older versions). ChatkitCore is for pure JVM, Chatkit is for Android.

Reply:
It might be clearer to say "non-Android"

Reply:
3b2075c58f0e611b57e4d8cca4f49f851d54badb

Reply:
Technically Android apps that prefer to use repositories instead of view models, and don't need push notifications, or don't want to pass us their context, can use chatkit-core. However we don't want to encourage it and for clarity it was good to make it clearer.